go.mod

```diff
@@ -4,10 +4,9 @@ go 1.12
 
 require (
 	bou.ke/monkey v1.0.2
-	github.com/Shopify/sarama v1.24.1
+	github.com/Shopify/sarama v1.27.2
```

Comment:
Is this update needed?

Reply:
Yes, the new feature implemented needs the update